code/src/utils/formatPrice.ts

```diff
@@ -0,0 +1,16 @@
+type FormatPriceOptions = {
+  currency?: "USD" | "EUR" | "GBP" | "BDT";
```

Comment:
What about separate type for currency? I like to move union types like variant into separate type

Reply:
That's what you mean? Refactored